module_oop/models.py

```diff
@@ -0,0 +1,100 @@
+import random
+import settings
+import exceptions
+
+
+class Enemy:
+    def __init__(self, level, health) -> None:
+        self.level = level
+        self.health = health
+
+    def decrease_health(self) -> None:
+        self.health -= 1
+        if self.health < 1: raise exceptions.EnemyDown
+
+
+    @staticmethod
+    def select_attack():
+        return random.randint(1, 3) # 1-Knight, 2-Thiev, 3-Wizart
+
+    @staticmethod
+    def select_defence() -> int:
+        return random.randint(1, 3)  # 1-Knight, 2-Thiev, 3-Wizart
+
+
+class Player:
+    def __init__(self, name, health, score, result_fight) -> None:
```

Comment:
![image](https://user-images.githubusercontent.com/22391544/233690274-8dacd295-d1d9-444e-9389-742a6b249c29.png)
```suggestion
    def __init__(self, name) -> None:
```